src/Symfony/Component/Notifier/Bridge/FakeSms/FakeSmsEmailTransport.php

```diff
@@ -54,7 +54,7 @@ public function supports(MessageInterface $message): bool
     }
 
     /**
-     * @param MessageInterface|SmsMessage $message
```

Comment:
Why this change?

Reply:
Change shouldn't be part of commit indeed. Reverted it.